src/components/CreateProfile/LocationPreferences/LocationPreferences.tsx

```diff
@@ -37,8 +41,9 @@ const LocationPreferences = () => {
           />
         </div>
         <div>
-          <InputFormError error={errors.city}>
+          <InputFormError error={errors.city} touched={touched.city}>
```

Comment:
**nitpick:** We don't need to pass `touched` here. You can just check it in `error field`, so it's:

```tsx
  <InputFormError error={touched.city && errors.city && errors.city}>
```